code/src/components/Form.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react'
+import FirstQuestion from './FirstQUestion'
+import SecondQuestion from './SecondQuestion'
+import ThirdQuestion from './ThirdQuestion'
+import FourthQuestion from './FourthQuestion'
+import FifthQuestion from './FifthQuestion'
+import Overview from './Overview'
+
+import { beer } from '../beer.js'
+
+const initialState = [
+    {
+      "name": "Lager",
+      "checked": false
+    },
+    {
+      "name": "Ale",
+      "checked": false
+    },
+    {
+      "name": "Sour",
+      "checked": false
+    },
+    {
+      "name": "Porter",
+      "checked": false
+    }
+  ]
+
+const Form = () => {
+  const [nameInput, setNameInput] = useState ('') 
+  const [emailInput, setEmailInput] = useState('')
+  const [addressInput, setAddressInput] = useState('')
+  const [brewery, setBrewery] = useState('')
+  const [step, setStep] = useState(1)
+  const [showResult, setShowResult] = useState(false)
+  const [selectedBeers, setSelectedBeers] = useState(initialState)
+
+  const handleOnChange = (beerType) => {
+    const updatedSelections = selectedBeers.map((beer) =>
+    beer.name === beerType ? { ...beer, checked: !beer.checked } : beer
+    )
+    setSelectedBeers(updatedSelections)
+  }
+
+  const onNameInputChange = (event) => {
+    setNameInput(event.target.value)
+  }
+
+  const onEmailInputChange = (event) => {
+    setEmailInput(event.target.value)
+  }
+
+  const onAddressInputChange = (event) => {
+    setAddressInput(event.target.value)
+  }
+
+  const onBreweryChange = (event) => {
+    setBrewery(event.target.value)
+  }
+
+  const onStepChange = (motion) => {
+    setStep(step + motion)
+  }
+
+  const onOverview = () => {
+    setShowResult(false)
+    setNameInput('')
+    setEmailInput('')
+    setAddressInput('')
+    setBrewery('')
+    setSelectedBeers(initialState)
+    setStep(1)
+  }
+
+  const onFinalQuestion = () => {
+    setShowResult(true)
+  }
+
+  return (
```

Comment:
I like how you added the ternary operator here for the Overview and the form questions.